### DISCUSSION TITLE
Cannot read property 'append' of null

### DISCUSSION THREAD

**Discussion by marcianos** [cypress-io/cypress#17856] [Component Testing]
Maybe somebody can help me. I start using Cypress and it is quite cool with some configuration overhead.
I want to test my own component and I use Vue and BootstrapVue.

I run my component test and got the error message:

`Cannot read property 'append' of null`

when trying to mount a component:

```js
import { mount } from '@cypress/vue'
import ReportCard from '..//src/components/ReportCard.vue'

describe('ReportCard component', () => {
  it('works', () => {
    mount(ReportCard)
  })
})
```

Can somebody help me with that?

**Comment by mmaietta** [comment]
@marcianos I'm running into this same issue on latest cypress 9.5.2.
Are you using webpack dev server + webpack 5 by chance? If so, do you have a custom configuration?
